### PR TITLE
Update package maintainers

### PR DESCRIPTION
**Pull request by gbiggs**

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <name>tinyxml_vendor</name>
   <version>0.8.2</version>
   <description>CMake shim over the tinxml library.</description>
-  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```
Comment (clalancette) [inline package.xml:7]: As a courtesy, we usually take the old maintainers and move them to an "author" tag to recognize their work in maintenance.  I'll suggest that here.

**Review by clalancette — CHANGES_REQUESTED** [review]
One minor change, then this will be good to merge.

File: package.xml (hunk 1 above)
Comment (gbiggs) [reply package.xml:7]: Done in ba8b977.

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
This was done elsewhere, so closing.

**Pull request closed without merging**